sql/core/src/main/scala/org/apache/spark/sql/execution/SparkQl.scala

```diff
@@ -370,13 +370,33 @@ private[sql] class SparkQl(conf: ParserConf = SimpleParserConf()) extends Cataly
       case Token("TOK_CLEARCACHE", Nil) =>
         ClearCacheCommand
 
+      // SHOW TABLES [(IN|FROM) database_name] ['identifier_with_wildcards']
```

Comment:
FYI to avoid conflicts with #12071 let's revert all the changes in this file since we're not using this parser anymore anyway.